.github/workflows/tests.yaml

```diff
@@ -25,6 +25,34 @@ jobs:
           cd api
           DJANGO_SECRET_KEY="testkey" python3 manage.py test
 
+  shellcheck:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

Comment:
Why `v2`? You are using `v3`in other places. I'd also pin to an exact version and let renovate do the updating

Reply:
Fixed and pinned to the latest version